client/src/components/Home.js

```diff
@@ -62,9 +57,9 @@ const Home = ({ user, logout }) => {
     });
   };
 
-  const postMessage = (body) => {
+  const postMessage = async (body) => {
     try {
-      const data = saveMessage(body);
+    const { data } = await axios.post('/api/messages', body);
```

Comment:
:100:, but apply the above changes and the code will be more modular. :slightly_smiling_face: